### ISSUE TITLE
2 Extra Mainland China Data

### ISSUE THREAD

**Issue by Peteryang0922** [CSSEGISandData/COVID-19#672]
Looks like all of the 'Mainland China' has been deleted, but there are still 2 Mainland China data remained in the updated data, which are Gansu and Hebei Province. Probably can delete these two.

Gansu | Mainland China | 2020-03-11T02:18:28 | 0 | 0 | 0 | 36.0611 | 103.8343
-- | -- | -- | -- | -- | -- | -- | --
  | Hebei | Mainland China | 2020-03-11T02:18:29 | 0 | 0 | 0 | 38.0428 | 114.5149

**Comment by JiPiBi** [comment]
You are right for the daily reports but in the time series the lines are all labelled China
